frontend/src/features/review/components/coverLetter/CoverLetterSection.tsx

```
@@ -0,0 +1,134 @@
+          data-modal='review'
+          className='fixed z-50'
+          style={{
+            top: selection.modalTop + SPACER_HEIGHT,
+            left: selection.modalLeft,
+          }}
+          onMouseDown={(e) => {
+            e.preventDefault();
+            e.stopPropagation();
+          }}
+        >
+          <ReviewModal
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)